### PR TITLE
[Processing] fix parameter to open the Retile algorithm

### PR DESCRIPTION
**Pull request by ghtmtt**
just a small fix to a parameter for the gdal Retile function. Nothing big at all

### CODE REVIEW

**Comment by m-kuhn** [comment]
Thanks @ghtmtt

File: python/plugins/processing/gui/MultipleInputPanel.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def setSelectedItems(self, selected):
         # No checking is performed!
         self.selectedoptions = selected
         self.leText.setText(
-            self.tr('%d elements selected') % len(self.selectedoptions))
+            self.tr('%d elements selected') % self.selectedoptions)
```
Comment (alexbruy) [inline python/plugins/processing/gui/MultipleInputPanel.py]: I'm afraid this is incorrect fix, as `self.selectedoptions` in the code defined as list

Comment (ghtmtt) [inline python/plugins/processing/gui/MultipleInputPanel.py]: really? With this fix the UI opens fine. Should we revert the commit?

Comment (m-kuhn) [inline python/plugins/processing/gui/MultipleInputPanel.py]: It's already reverted. What's the error you get?

**Pull request merged**